microsoft-edge/devtools-guide-chromium/customize/theme.md

```diff
@@ -0,0 +1,98 @@
+---
+description: How to apply different color themes to Microsoft Edge DevTools.
+title: Apply color themes to Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/22/2021
```

Comment:
Add an entry to the redirect file.
Deleted customize/dark-theme.md
Added customize/theme.md
Do like this example: Redirect to support PR1330 · Pull Request #1362
https://github.com/MicrosoftDocs/edge-developer/pull/1362

Reply:
Ah cool, I wasn't aware of this redirect file. Adding the entry now.